pkg/kafka_client/client.go

```diff
@@ -110,8 +152,7 @@ func (client *KafkaClient) ConsumerPull() (KafkaMessage, kafka.Event) {
 func (client KafkaClient) HealthCheck() error {
 	client.consumerInitialize()
 
-	topic := ""
-	_, err := client.Consumer.GetMetadata(&topic, false, 200)
+	_, err := client.Consumer.GetMetadata(nil, true, 2000)
```

Comment:
I am 2,500 miles from my Kafka cluster and the use SASL_SSL increases the number of round trips, so the 200mSec timeout was too short.  From my read of the GetMetadata, the nil and alltopics=true is the generic call.  I thought the topic := "" was an issue until I found the 200mSec was too short.  I left it this way as it seems more correct to me.

Reply:
It seems ok, but it can also be moved to the plugin configuration to get the desired timeout from the user. What do you think?

Reply:
Yeah, I thought about it.  I tried and failed and it was near the end of my day, so I just buttoned up what I had.  I'll try again this weekend and update this PR.